DefaultSettings.php

```diff
@@ -884,6 +889,7 @@
 		'smwgPropertiesCacheExpiry' => 3600,
 		'smwgStatisticsCache' => true,
 		'smwgStatisticsCacheExpiry' => 3600,
+		'api.browse' => 3600
 	),
```

Comment:
@mwjames Cannot this be named somewhat more consistent like e.g. `smwgApiBrowse`?

Reply:
I'm considering to rename the others because there is some cruft, maybe something like:

* `special.statistics` which can be either a number or `false` and would kill one of the settings
* `special.properties`
* `special.unusedproperties`
* `special.wantedproperties`

Reply:
Makes sense to me. Also this would indeed make configuration a bit easier. Go for it. :)